File: CS/WindowsApplication3/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace WindowsApplication3 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void chartControl1_ObjectHotTracked(object sender, DevExpress.XtraCharts.HotTrackEventArgs e) {
            // Display the object's type.
            this.Text = e.Object.GetType().ToString();
            // Display the additional object's type, when it is available.
            if (e.AdditionalObject != null) {
                this.Text += " (" + e.AdditionalObject.GetType().ToString() + ")";
            }
        }

        private void chartControl1_ObjectSelected(object sender, DevExpress.XtraCharts.HotTrackEventArgs e) {
            // Prevent the chart's diagram from being selected.
            if (e.Object is XYDiagram) {
                e.Cancel = true;
            }
            else {
                MessageBox.Show(e.Object.GetType().ToString());
            }
        }
    }
}

File: VB/WindowsApplication3/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace WindowsApplication3
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub chartControl1_ObjectHotTracked(ByVal sender As Object, ByVal e As DevExpress.XtraCharts.HotTrackEventArgs) Handles chartControl1.ObjectHotTracked
			' Display the object's type.
			Me.Text = e.Object.GetType().ToString()
			' Display the additional object's type, when it is available.
			If e.AdditionalObject IsNot Nothing Then
				Me.Text &= " (" & e.AdditionalObject.GetType().ToString() & ")"
			End If
		End Sub

		Private Sub chartControl1_ObjectSelected(ByVal sender As Object, ByVal e As DevExpress.XtraCharts.HotTrackEventArgs) Handles chartControl1.ObjectSelected
			' Prevent the chart's diagram from being selected.
			If TypeOf e.Object Is XYDiagram Then
				e.Cancel = True
			Else
				MessageBox.Show(e.Object.GetType().ToString())
			End If
		End Sub
	End Class
End Namespace